account/migrations/0041_merge_20200726_1740.py

```diff
@@ -0,0 +1,13 @@
+# Generated by Django 3.0.3 on 2020-07-27 00:40
```

Comment:
Can you fix these migration merge files? Rebase onto master, delete all the migration files you created (and these merge ones), drop your database, and then recreate your database and run makemigrations again.